python-package/xgboost/core.py

```diff
@@ -841,6 +841,45 @@ def feature_types(self, feature_types):
                 None,
                 c_bst_ulong(0)))
 
+    @property
+    def feature_weights(self):
```

Comment:
Consistency of the interface is really important. For this reason I don't think it should be defined as a property, but should take the same interface as row weights.

Reply:
@RAMitchell It's consistent with feature type and feature name ...  But sure it can look like row info, just writing a setter and getter as explicit function doesn't seem very pythonic.